account/account.py

```diff
@@ -20,6 +20,14 @@ class Meta:
 db.connect()
 db.create_tables([Account])
 
+ACCOUNT_PARAMETERS ={
+    '1': Account.account_number,
+    '2': Account.account_name,
+    '3': Account.balance,
+    '4': Account.user_id,
+    '5':Account.currency_id
```

Comment:
Czy poza nazwą i aktualizacją balansu, konto może zmienić numer, użytkownika i walutę? Tego w sumie nie ustaliliśmy dokładnie

Reply:
Póki co założyłam, że można zmienić wszystko poza ID. Założyłam, że jest to też furtka, jeśli ktoś się pomylił przy tworzeniu konta.

Reply:
Ok, ma sens. Wtedy ta furtka musiałaby być mocno pilnowana, żeby nie namieszać za dużo.

Reply:
Trzeba zatem przemyśleć upilnowanie tego. Póki co zostawiłabym to na przyszłość. Jak zostanie czas.